Description: Remove Policy and Update Desktop, Fix Current WiFi
 bash read /etc/Network is need sudo. nmcli is no need sudo.
 so remove policy file and update desktop file.
 fix current wifi get mode.
 .
 wifi-qr (0.1-1) unstable; urgency=medium
 .
   * Initial upload.
Author: kokoye2007 <kokoye2007@gmail.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: 2020-06-09

--- wifi-qr-0.1.orig/wifi-qr
+++ wifi-qr-0.1/wifi-qr
@@ -141,11 +141,14 @@ terminal_qr() {
   echo 
 }
 
+
 current_wifi_ssid() {
-	nmcli device wifi show-password
+  WIFIS="$(nmcli -t -f active,ssid dev wifi | egrep '^yes' | cut -d\: -f2 | tr -d '\r\n')"
+  echo -e "${BLUE}===========================================${RESET}"
+  echo -e "\nYou current active SSID is:${BOLD}${RED} $WIFIS\n ${RESET}"
+  echo -e "${BLUE}===========================================${RESET}"
 }
 
-
 #Data Extractor nmcli version
 call_wifi_pass() {
   SSID="$WIFIS"
@@ -374,6 +377,6 @@ case $1 in
 			q for QR Scan and Connect WiFi GUI
 			v for WiFi-QR Version is $VERSION
 		"
-  current_wifi_ssid
+  nmcli device wifi show-password
   ;;
 esac
--- wifi-qr-0.1.orig/wifi-qr.desktop
+++ wifi-qr-0.1/wifi-qr.desktop
@@ -3,7 +3,7 @@ Type=Application
 Version=0.1
 Name=WiFi QR
 Comment=Wifi share with QR Code
-Exec=sh -c 'pkexec wifi-qr g'
+Exec=sh -c 'wifi-qr g'
 Icon=wifi-qr
 Keywords="QR; WiFi; Password; Share;"
 Terminal=false
@@ -12,10 +12,10 @@ Actions=ScanQR;CreateQR;
 
 [Desktop Action ScanQR]
 Name=Scan WiFi QR Code
-Exec=sh -c 'pkexec wifi-qr q'
+Exec=sh -c 'wifi-qr q'
 Terminal=false
 
 [Desktop Action CreateQR]
 Name=Create WiFi QR Code
-Exec=sh -c 'pkexec wifi-qr c'
+Exec=sh -c 'wifi-qr c'
 Terminal=false
